x-pack/test/api_integration/apis/management/rollup/index_patterns_extensions.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+import querystring from 'querystring';
+
+import { registerHelpers } from './rollup.test_helpers';
+import { INDEX_TO_ROLLUP_MAPPINGS, INDEX_PATTERNS_EXTENSION_BASE_PATH } from './constants';
+import { getRandomString } from './lib';
+
+export default function ({ getService }) {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  const {
+    createIndexWithMappings,
+    getJobPayload,
+    createJob,
+    cleanUp,
+  } = registerHelpers({ supertest, es });
+
+  describe('index patterns extension', () => {
+    describe('Fields for wildcards', () => {
+      const BASE_URI = `${INDEX_PATTERNS_EXTENSION_BASE_PATH}/_fields_for_wildcard`;
+
+      describe('query params validation', () => {
+        let uri;
+        let body;
+        let params;
```

Comment:
Minor nit, but when I see variables defined outside of the scope of `it` and `test` calls, I expect them to be consumed or set outside of that scope too, e.g. set in a `beforeEach` or set in one test and then consumed in later ones. I think the code becomes simpler if these variables are defined as `const` within each test.